packages/@aws-cdk/assertions/test/template.test.ts

```diff
@@ -433,10 +435,10 @@ describe('Template', () => {
 
       const inspect = Template.fromStack(stack);
       const result = inspect.findOutputs('*', { Value: 'Fred' });
-      expect(result).toEqual([
-        { Value: 'Fred', Description: 'FooFred' },
-        { Value: 'Fred', Description: 'BarFred' },
-      ]);
+      expect(result).toEqual({
```

Comment:
Now that `findResources()` returns an object, would it make more sense to use `Match.objectLike()` or `Match.objectEquals()`? My concern is that the object order is arbitrary so conceivably `Bar` could come before `Foo` and that shouldn't result in a failed test.

Reply:
How about something like this instead? This should be object order insensitive

```ts
expect(Object.keys(result)).toEqual(2);
expect(result.Foo).toEqual(...);
expect(result.Bar).toEqual(...);
```